ext/oci8/oci8_interface.c

```diff
@@ -233,7 +236,8 @@ PHP_FUNCTION(oci_free_descriptor)
 		RETURN_THROWS();
 	}
 
-	if ((tmp = zend_hash_str_find(Z_OBJPROP_P(z_descriptor), "descriptor", sizeof("descriptor")-1)) == NULL) {
+	tmp = Z_OCILOB_DESCRIPTOR_P(z_descriptor);
```

Comment:
Based on `php_soap_deref` added in `https://github.com/php/php-src/commit/50484b59cd4ff5a1d32ce520fc7a7181b25ed0d9`, I'd guess that these properties could benefit from dereferencing (pls double check it)